src/averageAges.js

```diff
@@ -20,6 +20,11 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+  const mens = people.filter(({ sex, died }) => sex === 'm'
+    && (century ? century === Math.ceil(died / 100) : true));
+
+  return Math.round(((mens.reduce((sum, { born, died }) =>
+    sum + (died - born), 0) / mens.length) * 100)) / 100;
```

Comment:
This logic will be reused in next function too, let's make a separate helper function for it
Also better to move reduce result to constant